src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1367,8 +1367,6 @@ export default class AircraftModel {
                 break;
 
             case FLIGHT_PHASE.LANDING: {
-                this.target.heading = this.mcp.course;
```

Comment:
@felixscheffer could you elaborate on _why_ this one little change fixes this issue?  I think I know where you're going with this, but it's been probably over a year since I was in this part of the code.  So I'm a little fuzzy on what is happening here.

@erikquinn do you have any thoughts on this one?

Reply:
`updateTarget( `) calls  `_calculateTargetedHeadingDuringLanding() ` during  `FLIGHT_PHASE.LANDING ` to calculate a course correction to steer the aircraft towards the runway (which it does). Then,  `overrideTarget() ` overrides the new target heading with `mcp.course`. 

So in the end, `_calculateTargetedHeadingDuringLanding()` has no effect and the ` target.heading ` is always  `mcp.course` 

Maybe there is a scenario where the target heading should be `mcp.course` but IMHO it should be addressed in `_calculateTargetedHeadingDuringLanding()`

Reply:
Hm. Must have always been an issue then. A recent change made the phase change to `LANDING` occur further out, and that's why we're seeing an increase in the severity of them missing the runway. I concur with the removal :+1: